scss/popup.scss

```diff
@@ -4,6 +4,7 @@
 
 // Dependencies
 @import "common/all";
+@import "button";
```

Comment:
Where does the popup require a button?

Reply:
It's possible to have buttons in in the popup, namely in the filters and in toolbar. The latter is problematic, because the styles for button group tend to override those in popup. Reason being that button group is copied as is, rather than being transformed in the overflow popup menu

Reply:
Technically this should be handled by the component that uses the popup, but the action buttons are used in many popups across the suites. How about we include them directly?